app/views/planning_applications/review/tasks/_review_assessment.html.erb

```diff
@@ -44,12 +44,20 @@
     )%>
   <% end %>
   <% if @planning_application.application_type.planning_conditions? %>
-    <% if @planning_application.condition_set.present? && @planning_application.condition_set.current_review&.status != "not_started" %>
+    <% if @planning_application.condition_set&.current_review&.started? %>
```

Comment:
Just double checking - this link will still show if there aren't any conditions?

Reply:
I think it ought to be the same logic, just simplified by turning `status != "not_started"` into a method, and using `&.` to make the `present?` check unnecessary.

It'll not show if the condition set is nil, which I think is the same as before.